apps/frontend/src/app/5_pages/StakePage/components/StakingStatistics/StakingStatistics.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo } from 'react';
+
+import { t } from 'i18next';
+import { Trans } from 'react-i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+import {
+  HelperButton,
+  Link,
+  Paragraph,
+  ParagraphSize,
+  ParagraphStyle,
+} from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import {
+  APR,
+  TOKEN_RENDER_PRECISION,
+  VP,
+} from '../../../../../constants/currencies';
+import { MAX_STAKING_APR_LINK } from '../../../../../constants/links';
+import { translations } from '../../../../../locales/i18n';
+import { fromWei } from '../../../../../utils/math';
+import { GlobalStatRender } from '../../StakePage.utils';
+import { useGetStakingStatistics } from './hooks/useGetStakingStatistics';
+
+export const StakingStatistics = () => {
+  const { totalStakedSov, totalVotingPower, maxStakingApr } =
+    useGetStakingStatistics();
+
+  const renderMaxStakingApr = useMemo(
```

Comment:
I don't see "Max staking APR" but there are no errors in console, can you look into that?
<img width="1582" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/f83c7afb-da51-4a14-a7b6-c17794c2dc2a">

Reply:
yep, this is related to the CORS issue, on `sovryn.app` it should work (I added functionality for getting data from redash.sovryn.app and we need to add `REACT_APP_REDASH_API_KEY` with API_key value to the env file)
cc @creed-victor

Reply:
@pietro-maximoff reach out to devops regarding the CORS issue. 

Regarding `REACT_APP_REDASH_API_KEY` - I can show you how to add it to Netlify.